python/ray/tests/test_component_failures_3.py

```diff
@@ -84,6 +84,9 @@ def test_driver_lives_sequential(ray_start_regular):
     ray.worker._global_node.kill_monitor()
     ray.worker._global_node.kill_raylet_monitor()
 
+    if os.environ.get("RAY_GCS_SERVICE_ENABLED", None):
```

Comment:
Define this `RAY_GCS_SERVICE_ENABLED` as a constant since you use it many times.

Reply:
fixed